scripts/index.js

```diff
@@ -41,3 +41,349 @@ navListItem.forEach((listItem) => {
+  popup.appendChild(newPar);
+  popup.appendChild(popBtnCntnr);
+
+  return popup;
+}
+
+// function to create popup for mobile version
+function createDesktopPopup() {
+  const desktopPop = createMobilePopup();
+  desktopPop.classList.add('popup-desktop');
+  const dtheader = desktopPop.querySelector('.popup-header');
+  dtheader.classList.add('dt-pop-header');
+  const dtdescr = desktopPop.querySelector('.popup-description');
+  dtdescr.classList.add('dt-pop-descr');
+  const btnCntnr = desktopPop.querySelector('.pop-btn-contnr');
+  btnCntnr.classList.add('dt-btn-cntnr');
+  const dtliveBtn = desktopPop.querySelector('.live-button');
+  dtliveBtn.classList.add('dt-live-button');
+  const dtSrcBtn = desktopPop.querySelector('.src-button');
+  dtSrcBtn.classList.add('dt-src-button');
+
+  const projNav = document.createElement('div');
+  projNav.classList.add('proj-nav');
+  const leftProjBtn = document.createElement('button');
+  leftProjBtn.classList.add('proj-nav-left');
+  leftProjBtn.textContent = 'Prevoius project';
+  const rightProjBtn = document.createElement('button');
+  rightProjBtn.classList.add('proj-nav-right');
+  rightProjBtn.textContent = 'Next project';
+  projNav.append(leftProjBtn);
+  projNav.append(rightProjBtn);
+  desktopPop.append(projNav);
+  return desktopPop;
+}
+
+function createPopup(project) {
+  let popupwindow;
+  if (window.innerWidth <= 768) {
+    popupwindow = createMobilePopup();
+  }
+  if (window.innerWidth > 768) {
+    popupwindow = createDesktopPopup();
+  }
+  const container = document.querySelector('.projects-container');
+  container.appendChild(popupwindow);
+  loadNewProj(project);
+  document.body.style.overflow = 'hidden';
+}
+
+// create layout in mobile and desktop version
+
+const projectsContainer = document.querySelector('.projects-container');
+function createProjectTile(projectObj) {
+  const tile = document.createElement('div');
+  tile.classList.add('project-tile');
+  tile.id = `project-tile-${projectObj.projectno}`;
+  const thumbNail = document.createElement('img');
+  thumbNail.src = projectObj.image;
+  thumbNail.alt = projectObj.imageAlt;
+  thumbNail.classList.add('project-thumbnail');
+  tile.appendChild(thumbNail);
+
+  const details = document.createElement('div');
+  details.classList.add('project-details');
+
+  const projTitle = document.createElement('h3');
+  projTitle.classList.add('project-title');
+  projTitle.textContent = projectObj.name;
+
+  const techContainer = document.createElement('ul');
+  techContainer.classList.add('tech-item-list');
+
+  projectObj.technology.forEach((tech) => {
+    const techItem = document.createElement('li');
+    techItem.classList.add('tech-item');
+    techItem.textContent = tech;
+    techContainer.appendChild(techItem);
+  });
+  const projectButton = document.createElement('button');
+  projectButton.classList.add('button-generic', 'button-project');
+  projectButton.textContent = 'See this Project';
```

Comment:
@Favourezeugwa Hi Favour, I have attached a screenshot of how this part looks in my browser for a screen approximately 550px wide. Can i know at what resolution you were testing this please. I cannot reproduce this in my browser(firefox).

![Screenshot_20221221_215057](https://user-images.githubusercontent.com/116718155/208954525-b4d00973-a565-4c52-8c5c-b3d893ed5a69.png)

Reply:
That's alright @ram1117, I used 375px (ideal screen size for mobile screens). I guess there was a bug. I can see the `see this project` button now. This will be marked as optional. You can go ahead with the rest of the changes 👍🏽

Reply:
> @Favourezeugwa Hi Favour, I have attached a screenshot of how this part looks in my browser for a screen approximately 550px wide. Can i know at what resolution you were testing this please. I cannot reproduce this in my browser(firefox).
> 
> ![Screenshot_20221221_215057](https://user-images.githubusercontent.com/116718155/208954525-b4d00973-a565-4c52-8c5c-b3d893ed5a69.png)

Hello @ram1117 and @Favourezeugwa, I believe what @Favourezeugwa was right on making that observation because of the way @ram1117 is creating the modal. The desktop and the mobile modals are being created separately hence it is not working responsively. If you open the modal popup from a desktop screen width it maintains the same shape and dimensions without switching to the mobile popup modal for all screens of both desktop and mobile and vice versa.

@ram1117, kindly check and rectify this issue in the script.js file, lines 299 - 311 `createPopup` function. :wink: